app/routers/roles.py

```
@@ -0,0 +1,94 @@
+
+@router.get("/{role_id}", response_model=schemas.RoleResponse)
+def read_role(role_id: int, db: Session = Depends(get_db)):
+    """Get a specific role by ID"""
+    db_role = crud.get_role(db, role_id=role_id)
+    if db_role is None:
+        raise HTTPException(status_code=404, detail="Role not found")
+    return db_role
+
+
+@router.put("/{role_id}", response_model=schemas.RoleResponse)
+def update_role(role_id: int, role_update: schemas.RoleUpdate, db: Session = Depends(get_db)):
```

Ruff (0.13.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)